### ISSUE TITLE
Review review-requested, and normal changesets in iD

### ISSUE THREAD

**Issue by d4f5409d** [openstreetmap/iD#9417]
### Description

It would be cool to review if a changeset is correct right in iD, maybe we could even upvote/downvote changesets to check if it's appropriate.

**Comment by Zaczero** [comment]
Does OpenStreetMap data support such a feature? If not, adding it to iD alone wouldn't make much sense - I believe.

**Comment by Dimitar5555** [comment]
iD is an editor. It can't do QA on changesets or review them. The best it can do is validate the data that is loaded. [OSMCha](https://osmcha.org) allows you to mark a changeset as good or bad, flag bad features, add comments etc.

**Comment by d4f5409d** [comment]
> Does OpenStreetMap data support such a feature? If not, adding it to iD alone wouldn't make much sense - I believe.

Yes there's a feature in iD to request review. Right at uploading the changeset.

**Comment by d4f5409d** [comment]
> iD is an editor. It can't do QA on changesets or review them. The best it can do is validate the data that is loaded. [OSMCha](https://osmcha.org) allows you to mark a changeset as good or bad, flag bad features, add comments etc.

But why use a different tool? Why not integrate all of useful features in one?

**Comment by Dimitar5555** [comment]
> But why use a different tool? Why not integrate all of useful features in one?

Because it's not practical. It's better to have more specialised and sophisticated programs than to have less but more general purpose ones. That way developers can focus on what they can do the best, instead of each one trying to reinvent the same wheel.

**Comment by Zaczero** [comment]
> Yes there's a feature in iD to request review. Right at uploading the changeset.

But this simply add tags to a changeset. I am talking about you uploading your up/down votes to a specific changeset, and not write in the comments (which are a similar kind of system) (a naive approach would simply post comments "I upvote this changeset" - but that's not desirable). If such feature is not implemented on OpenStreetMap data side, this should be a priority first. I doubt that we want to setup external services for iD just to support up/down votes. It's a better idea to standardize it in some sort of way.

**Comment by tyrasd** [comment]
As others have mentioned above: this is more an issue regarding the [osm website](https://github.com/openstreetmap/openstreetmap-website), not specifically iD. It's probably best to start discussing this for example on the community forum?

(closing as "not planned" only because this is not directly related to iD)
